app/routes.php

```diff
@@ -41,4 +41,31 @@ function( Application $app, $pageName ) use ( $ffFactory ) {
 	}
 );
 
+$app->get(
+	'check-iban',
+	function( Request $request ) use ( $app, $ffFactory ) {
+		$useCase = $ffFactory->newValidateBankDataUseCase();
+		$responseModel = $useCase->checkIban( $request->get( 'iban', '' ) );
+
+		return $app->json(
+			$responseModel ?
+				[ 'status' => 'OK' ] + $responseModel->getBankData() :
+				[ 'status' => 'ERR' ]
+		);
+	}
+);
+
+$app->get(
+	'generate-iban',
+	function( Request $request ) use ( $app, $ffFactory ) {
+		$useCase = $ffFactory->newValidateBankDataUseCase();
+		$responseModel = $useCase->generateIban(
+			$request->get( 'accountNumber', '' ),
+			$request->get( 'bankCode', '' )
+		);
```

Comment:
"Check IBAN" and "Generate IBAN" are two different use cases, each with their own request model and response model. So a GenerateIbanUseCase and GenerateIbanRequest class are needed. (I've send a mail with some diagrams)